check/setup_test.go

```diff
@@ -273,6 +273,11 @@ func Test_deleteTopic_WhenCreateDeleteNodeFails_ReturnsError(t *testing.T) {
 
 	connection.EXPECT().DeleteTopic("health-check", gomock.Any()).Return(errors.New("New error"))
 
+	// here we are also testing that retriers do 3 tries before return an error
```

Comment:
can we actually use a cycle here taking advantage of this:
```
func newTestCheck() *HealthCheck {
	actionRetrierConfig := &ActionRetrierConfig{
		NumOfRetries: 3,
		Amount:       1 * time.Millisecond,
	}
```
so that we don't have to hardcode the number of attempts?